src/nls/fi/strings.js

```diff
@@ -459,7 +459,7 @@ define({
     "PROJECT_SETTINGS_TITLE"               : "Projektin asetukset kohteelle: {0}",
     "PROJECT_SETTING_BASE_URL"             : "Esikatselun perus-URL",
     "PROJECT_SETTING_BASE_URL_HINT"        : "Käytä paikall. palvelinta antamalla URL, kuten http://localhost:8000/",
-    "BASEURL_ERROR_INVALID_PROTOCOL"       : "Esikatselu ei tue {0}-protokollaa. Käytä joko http: tai https: -mallia.",
+    "BASEURL_ERROR_INVALID_PROTOCOL"       : "Esikatselu ei tue {0}-protokollaa. Käytä joko http: tai https: \u2011muotoa.",
```

Comment:
\u2011 was changed to `-` in other parts of this PR, maybe this should stay like so too? 🤔